frontend/src/utils/format_column_data.ts

```
@@ -23,18 +11,24 @@ export class GraphData {
   max_height: number;
   canvas: CanvasRenderingContext2D;
   root: string;
-  usedColors: Set<string> = new Set();
-  drawable_shapes: null = null;
+  drawable_shapes: DrawableShape[] = [];
+  canvas_container: HTMLCanvasElement;
+  current_radius: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)